airflow/api/client/json_client.py

```diff
@@ -40,63 +55,99 @@ def _request(self, url, method="GET", json=None):
             except Exception:
                 data = {}
             raise OSError(data.get("error", "Server error"))
-
         return resp.json()
 
     def trigger_dag(self, dag_id, run_id=None, conf=None, execution_date=None, replace_microseconds=True):
+        """Trigger a DAG run.
```

Comment:
Any reason for not 'typing' this method?

Reply:
Is it okay?

Reply:
I would prefer it's typed. No strong opinion though

Reply:
Do you mean method's arguments?